changelog/339.txt

```diff
@@ -0,0 +1,4 @@
+```release-note:breaking
```

Comment:
Instead of `breaking`, I think this would just be `bug`. That'll group it into the right place in our changelog whenever that gets generated. And, then we can probably get rid of the bug prefix on line 2.

Reply:
I...am so sorry about how irresponsible I am when copying and repurposing changelog files. 😢 plz forgive

Reply:
I forgive you! 😆